src/character.cpp

```diff
@@ -195,10 +195,10 @@ double Character::aim_per_move( const item& gun, double recoil ) const
     }
 
     // get fastest sight that can be used to improve aim further below @ref recoil
-    int cost = INT_MAX;
+    int sight_cost = INT_MAX;
     int limit = 0;
     if( !gun.has_flag( "DISABLE_SIGHTS" ) && effective_dispersion( gun.type->gun->sight_dispersion ) < recoil ) {
-        cost  = std::max( std::min( gun.volume() / 250_ml, 8 ), 1 );
```

Comment:
Balance aside, this is a bugfix.  The previous version based aiming speed for unmodded guns on gun size, then completely ignored it once a sight ws added.